src/go/MONIT/ggus_parser.go

```diff
@@ -291,25 +298,88 @@ func processResponse(url, format, accept, out string) {
 		}
 	}
 
+	return resp
+}
+
+func retrieveTicketDetails(ticketId int) (string, []TicketComment) {
+	url := fmt.Sprintf("https://ggus.eu/?mode=ticket_info&ticket_id=%d", ticketId)
+	resp := ggusRequest(url, "text/html")
+	fmt.Println(resp)
+
+	doc, _ := html.Parse(resp.Body)
```

Comment:
Panos, please don't skip error, if parser will fail it is better to know why. I suggest to get the error, print it back and return. You may also consider returning error to upstream code, but it will require to change function definition.